todo-app/public/index.css

```diff
@@ -0,0 +1,31 @@
+h2{
```

Comment:
A few formatting issues in your CSS here. It's OK to indent with 2 spaces or 4 spaces, but choose one and stick to it. Also, add spaces after your class names and before the curly brackets.